### ISSUE TITLE
tsc exited with 2

### ISSUE THREAD

**Issue by kapilpipaliya** [Alorel/rollup-plugin-fast-tsc#2]
when i tried to use this plugin like:

```
import {fastTscPlugin} from "rollup-plugin-fast-tsc"
// in plugins:
fastTscPlugin({})

```
it says:
```
[!] (plugin fast-tsc) Error: tsc exited with 2
Error: tsc exited with 2
```

**Comment by Alorel** [comment]
Hey, this plugin was a buggy implementation attempt of another typescript compiler, written before I knew how to write plugins properly. It only exists because `@rollup/plugin-typescript` was (possibly still is) super memleaky, but I've since discovered [rollup-plugin-typescript2](https://www.npmjs.com/package/rollup-plugin-typescript2) which I'd recommend using instead - it's far better than any alternative I've found or tried to write. :slightly_smiling_face: 

Thanks for the reminder to archive this repo, though!
